src/components/Swiper/item.tsx

```diff
@@ -0,0 +1,5 @@
+const Item = ({ children }: React.PropsWithChildren<{ children: React.ReactNode }>) => {
+  return <div className="h-full w-full">{children}</div>;
+};
+
+export default Item;
```

Comment:
is this needed? isn't it the same as using div?

Reply:
The interaction is not as expected, but it can be left to be expanded later.